### PR TITLE
[refine](pipeline) refine some some operator close function

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

File: be/src/pipeline/exec/aggregation_sink_operator.cpp [hunk 1]
```diff
@@ -97,11 +97,11 @@ Status AggSinkLocalState::open(RuntimeState* state) {
         RETURN_IF_ERROR(
                 p._probe_expr_ctxs[i]->clone(state, Base::_shared_state->probe_expr_ctxs[i]));
     }
-    Base::_shared_state->agg_profile_arena = std::make_unique<vectorized::Arena>();
+    _agg_profile_arena = std::make_unique<vectorized::Arena>();
```
Comment (yiguolei) [inline be/src/pipeline/exec/aggregation_sink_operator.cpp:100]: 为什么要在open里？不是在构造函数里？

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run External

**Comment by Mryange** [comment]
run external

**Comment by Mryange** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]

**Review by yiguolei — APPROVED** [review]

**Pull request merged**